tests/scripts/create_oidc_identities

```diff
@@ -1,6 +1,8 @@
 #! /usr/bin/env bash
 set -e
 
+sudo lxc config set user.show_permissions=true
```

Comment:
```suggestion
lxc config set user.show_permissions=true
```
We probably don't need the sudo here, as the other commands below don't have it either?

Same for the delete script.